### PR TITLE
📖 Preliminary release dates for v1.5

### PR DESCRIPTION
**Pull request by ykakarap**
**What this PR does / why we need it**:

This PR adds the tentative release dates for the v1.5.0 release cycle.

Thanks to @sbueringer for coming up with the dates 🙏🏼.

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

### CODE REVIEW

**Comment by ykakarap** [comment]
cc @kubernetes-sigs/cluster-api-release-team

File: docs/release/releases/release-1.5.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+| v1.5.0-rc.1 released                                 | Release Lead | Tuesday 18th July 2023     | week 16  |
+| **v1.5.0 released**                                  | Release Lead | Tuesday 25th July 2023     | week 17  |
+| *v1.3.x & v1.4.x released*                           | Release Lead | Tuesday 25th July 2023     | week 17  |
+| Organize release retrospective                       | Release Lead | TBC                        | week 17  |
+
+After the `.0` release monthly patch release will be created.
+
+## Release team
+
+TODO
```
Comment (cpanato) [inline docs/release/releases/release-1.5.md:29]: can maybe add a description in what need to be done here?

**Review by cpanato — COMMENTED** [review]
lgtm

**Comment by sbueringer** [comment]
@ykakarap In case you were not planning to do it anyway, can you bring this up in the office hours today?

File: docs/release/release-cycle.md [hunk 2]
```diff
@@ -3,7 +3,6 @@
 The release cycle is the time between when we start working on a release on the `main` branch until the `.0` release (e.g. `v1.3.0`) is released.
 
 **Note**: For the `v1.3` and `v1.4` releases we assume each release cycle will last approximately 4 months (~ 17 weeks).
-The release cycle duration will be revisited after the `v1.4.0` release.
```
Comment (sbueringer) [inline docs/release/release-cycle.md:6]: Let's please change this sentence to express that we will use 4 month cadence for 2023 and then revisit 2024.

File: docs/release/release-cycle.md [hunk 3]
```diff
@@ -3,7 +3,6 @@
 The release cycle is the time between when we start working on a release on the `main` branch until the `.0` release (e.g. `v1.3.0`) is released.
 
 **Note**: For the `v1.3` and `v1.4` releases we assume each release cycle will last approximately 4 months (~ 17 weeks).
```
Comment (furkatgofurov7) [inline docs/release/release-cycle.md:5]: do we want to add new v1.5 release here as well? Then we have to update the same later at the end of the doc where we refer to  https://github.com/kubernetes-sigs/cluster-api/blob/main/docs/release/release-cycle-overview.png. I wonder if this document should be generic and not mention at all release versions

Comment (sbueringer) [reply docs/release/release-cycle.md:5]: I think we should just write that we assume this cycle for 2023 and re-evaluate 2024 (xref: https://github.com/kubernetes-sigs/cluster-api/pull/8148#discussion_r1114103614)

Apart from that we don't have to mention specific releases in my opinion

Comment (sbueringer) [reply docs/release/release-cycle.md:5]: I think the most part of the doc should be very independent of specific release and only mention them as examples.

And then we have the question if we want to maintain the diagram going forward, if we keep it as an example or if we drop it. I think that's a question for the com team (cc @oscr).

File: docs/release/release-cycle.md (hunk 2 above)
Comment (ykakarap) [reply docs/release/release-cycle.md:6]: done.

**Review by CecileRobertMichon — COMMENTED** [review]
/lgtm

**Comment by fabriziopandini** [comment]
/lgtm
Thanks for driving this @ykakarap !

File: docs/release/release-cycle.md (hunk 3 above)
Comment (oscr) [reply docs/release/release-cycle.md]: We discussed this diagram during the comms meeting. We would like to suggest keeping the diagram as is for now. But to also create an improvement task to implement it as code and add it as a task for the release team to update. We feel the diagram is helpful for users and should maybe even be made more prominent. It clearly shows the great effort to keep multiple versions supported and when the end of life is for these versions.

cc @joekr @VibhorChinda @sayantani11

Comment (sbueringer) [reply docs/release/release-cycle.md]: Sounds good!

Comment (joekr) [reply docs/release/release-cycle.md]: @sbueringer do you by chance have a version that is editable? If not we can start to recreate it with other tools.

**Review by vincepri — APPROVED** [review]
/approve

File: docs/release/release-cycle.md (hunk 3 above)
Comment (sbueringer) [reply docs/release/release-cycle.md]:
```md
<!-- The release cycle png can be opened and edited in draw.io -->
![Release Cycle Overview](release-cycle-overview.png)
```

Just open the png directly in draw.io :)

**Pull request merged**